design-documents/graph-ql/coverage/requisitionList.graphqls

```diff
@@ -121,25 +121,29 @@ type Mutation {
     ): UpdateRequisitionListItemsOutput @doc(description: "Update Items in requisition list")
 
     addRequisitionListItemToCart(
-        requisition_list_id: ID!, @doc(description: "unique Id of requisition list")
-        item_ids: [ID!]! @doc(description: "selected requisition list items that are to be added")
+        requisitionListId: ID!, @doc(description: "unique Id of requisition list")
+        itemIds: [ID!]! @doc(description: "selected requisition list items that are to be added")
     ): AddRequisitionListItemToCartOutput @doc(description: "Add Requisition List Items To Customer Cart")
 
     copyItemsBetweenRequisitionList(
-        source_id: ID!, @doc(description: "unique Id of source requisition list")
-        destination_id: ID,  @doc(description: "unique Id of destination requisition list") # If null new requisition list will be created
-        item_ids: [ID!]! @doc(description:  "selected requisition list items that are to be copied from source")
+        sourceId: ID!, @doc(description: "unique Id of source requisition list")
+        destinationId: ID,  @doc(description: "unique Id of destination requisition list") # If null new requisition list will be created
+        itemIds: [ID!]! @doc(description:  "selected requisition list items that are to be copied from source")
     ): CopyItemsFromRequisitionListOutput @doc(description: "Copy Items from Requisition List to another requisition list")
 
     moveItemsBetweenRequisitionList(
-        source_id: ID!, @doc(description: "unique Id of source requisition list")
-        destination_id: ID, @doc(description: "unique Id of destination requisition list") # If null new requisition list will be created
-        item_ids: [ID!]! @doc(description: "selected requisition list items that are to be moved from source")
+        sourceId: ID!, @doc(description: "unique Id of source requisition list")
+        destinationId: ID, @doc(description: "unique Id of destination requisition list") # If null new requisition list will be created
+        itemIds: [ID!]! @doc(description: "selected requisition list items that are to be moved from source")
     ): MoveItemsFromRequisitionListOutput @doc(description: "Move Items from Requisition List to another requisition List")
+
+    clearCustomerCart(
+	cartId: String! @doc(description: "masked Cart Id")
+    ): ClearCustomerCartOutput @doc(description: "Clears the cart items")
 }
 
 type RemoveRequisitionListItemsOutput {
-    requisition_list : RequisitionList
+    requisitionList : RequisitionList
```

Comment:
Only arguments should be camel case. Fields should be underscored.
****
```suggestion
    requisition_list : RequisitionList
```

Reply:
@paliarush fixed it. Thanks